cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -117,16 +123,24 @@ createSPOGenesisAndFiles testnetOptions startTime (TmpAbsolutePath tempAbsPath')
       . HM.insert "updateQuorum"           (toJSON @Int 2)
       )
 
+  -- TODO: create-testnet-data should have arguments for
+  -- Alonzo and Conway genesis that are optional and if not
+  -- supplised the users get a default
```

Comment:
FYI this is tracked here: https://github.com/IntersectMBO/cardano-cli/issues/548 so it'll happen at some point